.github/workflows/medusa.yml

```diff
@@ -0,0 +1,26 @@
+name: Run medusa
+on:
+  push:
+    branches: [main, dev]
+  pull_request:
+    branches:
+      - "**"
+jobs:
+  tests:
+    concurrency: ci-medusa-${{ github.head_ref || github.run_id }}
+    runs-on: ubuntu-latest
+    container:
+      image: trailofbits/eth-security-toolbox:nightly
```

Comment:
Do you know if there is any delay before new Medusa releases are added to the image?

Reply:
idk about medusa, probably not, but foundry is currently at a version from july in the image